tests/integration/test_checks.py

```diff
@@ -0,0 +1,234 @@
+"""Integration tests for methods implemented on Check* classes."""
+import datetime
+
+import dateutil
+import pytest
+
+import github3
+
+from .helper import IntegrationHelper
+
+
+class TestCheckSuite(IntegrationHelper):
+    def get_repo(self, repository="westphahl/github3.py"):
```

Comment:
Should the cassettes be re-recorded with the official github3.py repo?

Reply:
They can be, but it's not entirely necessary.